api/capi/src/nnstreamer-capi-util.c

```diff
@@ -85,13 +85,13 @@ ml_tensors_info_destroy (ml_tensors_info_h info)
 {
   ml_tensors_info_s *tensors_info;
 
+  if (!info)
+    return ML_ERROR_NONE;
+
   check_feature_state ();
 
   tensors_info = (ml_tensors_info_s *) info;
 
-  if (!tensors_info)
-    return ML_ERROR_INVALID_PARAMETER;
-
   ml_tensors_info_free (tensors_info);
   g_free (tensors_info);
```

Comment:
how about setting these to null after free? (same for ml_tensors_data_destroy)

Reply:
I added it. please check it is your intention.

Reply:
Yes